<?xml version="1.0"?>
<launch>

    <rosparam param="/uwb/nodesId"> [100, 101, 102, 103, 204]</rosparam>
    <rosparam param="/uwb/nodesPos">[3, 3, 0.65, 3, -3, 0.25, -3, -3, 0.25, -3, 3, 0.25, 0, 0, 1.0]</rosparam>

    <include file="$(find hector_quadrotor_demo)/launch/outdoor_flight_gazebo.launch"/>
    <include file="$(find hector_quadrotor_teleop)/launch/xbox_controller.launch"/>
    
    <node respawn="false" pkg="uwb_driver" type="uwb_driver_gazebo_node" name="uwb_driver_gazebo_node" output="screen">
		<remap from="/ground_truth/pose" to="/ground_truth_to_tf/pose"/>
    </node>
  
</launch>
